hsrb_bringup/launch/hsrb_bringup.launch

```diff
@@ -17,7 +17,6 @@
 
   <include file="$(find hero_bringup)/hsrb_bringup/launch/include/hsrb_controller_common.xml">
     <arg name="namespace" value="$(arg namespace)" />
```

Comment:
If we don't use the ```use_laser_odom``` parameter, we don't have to set it at the top

Reply:
fixed.